dysnix/geth/values.yaml

```diff
@@ -265,3 +265,42 @@ config:
   pprof:
     enabled: false
     port: 6061
+
+### S3 directory structure
+### Currenly tested only 1 baseUrl per 1 node deployment
+###
+###   baseUrl
+###   ├── ancient_url
+###   └── chaindata_url
+###
+###   cat ancient_url   ->  baseUrl/<your_ancient_destination>
```

Comment:
It's unclear for me how to populate this metadata. WDYT on adding a complete example with [almost] real data ?

Reply:
I think we will use README for that